ASS/AWS.py

```diff
@@ -18,18 +19,25 @@ def set_logger(self, logger):
             raise Exception("Not a valid logger object")
 
     def empty_bucket(self, bucket):
+        s3client = boto3.client('s3', region_name= self.get_region())
+        bucket_name = bucket['Name']
+        versioning_status = s3client.get_bucket_versioning(Bucket=bucket_name)
         try:
-            self.logger.info(f"Connect to bucket {bucket}")
+            self.logger.info(f"Connect to bucket {bucket_name}")
             s3 = boto3.resource('s3')
-            bucket = s3.Bucket(bucket)
-            self.logger.info(f"Start deletion of all objects in bucket {bucket}")
+            bucket = s3.Bucket(bucket_name)
+            self.logger.info(f"Start deletion of all objects in bucket {bucket_name}")
             bucket.objects.all().delete()
-            self.logger.info(f"Finished deletion of all objects in bucket {bucket}")
+            if versioning_status['Status'] == "Enabled" or versioning_status['Status'] == "Suspended":
```

Comment:
This test is always `True`, `versioning_status` is always `Enabled` or `Suspended`.